serverless/serverless
# Use pythonX instead of pythonX.Y in invoke local

## What did you implement:
Closes #5115


## How did you implement it:
Runtime is either `python2.7` or `python3.6` and this changes invoke local to call `python2` or `python3` instead of the fully qualified version. This is necessary because py3k's `venv` module(unlike the older 3rd party `virtualenv` module) doesn't create a `pythonX.Y` executable/symlink.


## How can we verify it:
Create a python3.6 project, create&activate a virtualenv using `python3 -m venv && . ./venv/bin/activate` then install a module in the virtualenv (`pip install dateutil`) and import it in `handlers.py` (`import dateutil`). The current version will fail, this pr should pass


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
